frames/section.lua

```diff
@@ -239,6 +242,74 @@ local function onTitleClickOrDrop(section)
   events:SendMessage('bags/FullRefreshAll')
 end
 
+---@param section Section
+local function onTitleRightClick(section)
```

Comment:
Can you make this a method on Section and not a local function? Part of the ethos of BetterBags is to expose things as an API where possible.

Capitalize the function name when you expose it. Thanks!

Reply:
Sure, do you want the same thing for onTitleClickOrDrop ? Seems inconsistent.

Reply:
Yes, please! Thank you for catching that!